src/shared/version.go

```diff
@@ -1,5 +1,5 @@
 package shared
 
 const (
-	HyperdriveVersion string = "0.3.0-dev"
+	HyperdriveVersion string = "0.2.5-dev"
```

Comment:
This should go back to 0.3.0 as that's main now.

Reply:
yeah, let me change that when this PR is ready to go in. I still have to add a few more contract calls.